app/src/main/java/com/example/othregensburg/zapp/BarlistActivity.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.othregensburg.zapp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.google.gson.GsonBuilder
+import kotlinx.android.synthetic.main.activity_barlist.*
+import okhttp3.*
+import java.io.IOException
+
+class BarlistActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_barlist)
+
+        recyclerView_barlist.layoutManager = LinearLayoutManager(this)
+
+        fetchJson()
+    }
+
+
+    private fun fetchJson() {
+
+        //println("Try to fetch JSON")
```

Comment:
println is usually not used within Android development. You can log anything to Logcat with help of the Log class. Also: Better than this approach is to set a breakpoint and go into debugging the application. This will save you a lot of time in the end.

Anyways, when merging code back to develop such Log files should no more be in code.

Reply:
changed to logcat. I left it here because we are going to develop additional features to the feature